tasks/steps/blocks/update_package.yml

```diff
@@ -0,0 +1,54 @@
+---
+
+- name: 'Get package meta info'
+  any_errors_fatal: true
+  cartridge_set_package_info:
+    package_path: '{{ delivered_package_path }}'
+    app_name: '{{ cartridge_app_name }}'
+
+- name: 'Set "cartridge_app_name" from package info'
+  set_fact:
+    cartridge_app_name: '{{ package_info.name }}'
+
+- name: 'BLOCK : Enable Tarantool repository'
+  vars:
+    version: '{{ package_info.tnt_version }}'
+  when: cartridge_enable_tarantool_repo and package_info.tnt_version
+  block:
+    - name: 'Get repository setup script'
+      get_url:
+        url: https://tarantool-io.hb.bizmrg.com/website-static/tarantool/downloads/tarantool_installer.sh
+        dest: '/tmp/tarantool-installer.sh'
```

Comment:
I think that we can remove this script after installation.

Reply:
Maybe, but it's better to do it in another merge request

Reply:
Ok. Please file an issue and feel free to resolve conversation.